support-frontend/assets/pages/[countryGroupId]/student/components/StudentHeader.tsx

```diff
@@ -8,32 +8,38 @@ import type {
 import type { GeoId } from 'pages/geoIdConfig';
 import buildCheckoutUrl from '../helpers/buildCheckoutUrl';
 import type { StudentDiscount } from '../helpers/discountDetails';
-import LogoUTS from '../logos/uts';
 import {
 	cardContainer,
 	containerCardsAndSignIn,
 	heading,
 	headingWrapper,
 	subHeading,
-	universityBadge,
 } from './StudentHeaderStyles';
 import StudentPrice from './StudentPrice';
 import StudentProductCard from './StudentProductCard';
 
+interface StudentHeaderProps {
+	geoId: GeoId;
+	productKey: ActiveProductKey;
+	ratePlanKey: ActiveRatePlanKey;
+	landingPageVariant: LandingPageVariant;
+	studentDiscount: StudentDiscount;
+	headingCopy: string;
+	subheadingCopy: React.ReactNode;
```

Comment:
Normally I use `JSX.Element` does this enable the use of string as well as a component? Maybe I should use this always instead?

Reply:
Yeah `React.ReactNode` is broader (including string, basically anything React can render) so I went for that for the `subheadingCopy`. Actually this comment made me realise that `universityBadge` doesn't need to be so broad so I'll update that to `JSX.Element` 👍